x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/searchablesnapshots/SearchableSnapshotShardStats.java

```diff
@@ -137,17 +135,17 @@ public int hashCode() {
         private final Counter cachedBytesRead;
         private final TimedCounter cachedBytesWritten;
         private final TimedCounter directBytesRead;
+        private final TimedCounter optimizedBytesRead;
 
-        public CacheIndexInputStats(String fileName, long fileLength, long openCount, long innerCount, long closeCount,
+        public CacheIndexInputStats(String fileName, long fileLength, long openCount, long closeCount,
                                     Counter forwardSmallSeeks, Counter backwardSmallSeeks,
                                     Counter forwardLargeSeeks, Counter backwardLargeSeeks,
                                     Counter contiguousReads, Counter nonContiguousReads,
                                     Counter cachedBytesRead, TimedCounter cachedBytesWritten,
-                                    TimedCounter directBytesRead) {
+                                    TimedCounter directBytesRead, TimedCounter optimizedBytesRead) {
             this.fileName = fileName;
             this.fileLength = fileLength;
             this.openCount = openCount;
-            this.innerCount = innerCount;
```

Comment:
The "inner open count" information is obsolete since #53860 and has been removed